ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/RemoteConnectorExecutor.java

```diff
@@ -98,6 +98,8 @@ default void setModelRateLimiter(TokenBucket modelRateLimiter) {}
 
     default void setUserRateLimiterMap(Map<String, TokenBucket> userRateLimiterMap) {}
 
+    default void initialize() {}
```

Comment:
Seems this method is not being used, delete it?

Reply:
removed.